Doc/reference/datamodel.rst

```diff
@@ -925,8 +925,9 @@ Internal types
       the first line number of the function; :attr:`co_lnotab` is a string
       encoding the mapping from bytecode offsets to line numbers (for details
       see the source code of the interpreter); :attr:`co_stacksize` is the
-      required stack size (including local variables); :attr:`co_flags` is an
-      integer encoding a number of flags for the interpreter.
+      largest size, stack will ever grow to (after certain operations);
```

Comment:
Let's get rid of the comma here: "is the largest size the stack will ever grow to"

and maybe reword "after certain operations" to "due to certain instructions" or maybe even omit that part.